### DISCUSSION TITLE
Web3 custom auth testnet broken

### DISCUSSION THREAD

**Discussion by davedefiq** [Web3Auth/Web3Auth#681] [Help]
We use CustomAuth flutter sdk and started receving error messages like the below starting today - likely do to something using infura testnet under the hood, which has deprecated Rinkeby, Ropsten, and Kovan as of today (Oct. 5, 2022)

"error": {
    "code":-32601,
    "message":"Network decommissioned, please use Goerli or Sepolia instead",
    "data": {
     "see":"https://blog.infura.io/post/deprecation-timeline-for-rinkeby-ropsten-and-kovan-testnets"
    }
 }

**Comment by yashovardhan** [comment]
Hey @davedefiq 

Thanks for this query. We have received the notification by Infura about their Ropsten node being shut down. Our team is working towards moving the testnet to another node provider. In the meantime please use the mainnet/ cyan network for testing purposes.

**Reply by how2945ard** [reply]
Does this issue also affect idToken from getUserInfo?

I am only getting empty strings.


```
      > const userInfo = await web3auth.getUserInfo();
      > console.log(userInfo.idToken);
      "" (empty string)
```

**Reply by YZhenY** [reply]
Track updates here https://github.com/orgs/Web3Auth/discussions/685
